storage/indexer.go

```diff
@@ -43,6 +46,7 @@ func NewIndexer(storageClient *storage.Client, options IndexerOptions) *Indexer
 	return &Indexer{
 		storageClient: storageClient,
 		options:       options,
+		label:         "StorageIndexer",
```

Comment:
Hm.. could it be a const instead of a structure field?

Reply:
I wanted to follow the same structure that it is used for FileSystemIndexers ([link](https://github.com/elastic/package-registry/blob/main/packages/packages.go#L57)) where the label is part of the struct.